tests/test_experiment_server.py

```diff
@@ -56,6 +56,13 @@ def test_gunicorn_worker_config(self, webapp, active_config):
                 server.load_user_config()
                 assert server.options["workers"] == u"2"
 
+    def test_flask_secret_loaded_from_environ(self, webapp):
+        import os
+
+        os.environ["FLASK_SECRET_KEY"] = "A TEST SECRET KEY"
```

Comment:
I'm not sure if there's a danger of cross-test side-effects by actually changing environment variables, but I've noticed pytest suggests avoiding it:
https://docs.pytest.org/en/5.4.3/monkeypatch.html#monkeypatching-environment-variables

(I also suspect this is no different from using `mock`.)

Reply:
Right, at first I used `mock`, but ran into some non-mock related issues and tried to simplify. I should probably bring mock back, though it's really unlikely to matter with this particular variable.

Reply:
The only reason I think it might matter here is that someone doing something in the future where it *did* matter might copy the example.

Reply:
I've switched all tests to use `mock` to manage `os.environ` settings (by making the `env` fixture use `mock` and using that fixture in a few additional fixtures/tests that make use of `os.environ`).